### ISSUE TITLE
MigrationMap unused?

### ISSUE THREAD

**Issue by jouke** [fullcube/loopback-component-migrate#8]
While working on the pull request I just submitted, I noticed that the MigrationMap model seems unused. Apart from the index.js defining the model and the files that define the model, there is absolutely no reference to it.

Can it be removed? I'd rather not have unused tables in my database... :)

Cheers,

Jouke

**Comment by alireza-ahmadi** [comment]
I also agree with @jouke, MigrationMap seems unused and even [it's unit tests](https://github.com/fullcube/loopback-component-migrate/blob/722cab5af48a02fa6d5704e39cd39a9292a5c1ce/test/test.js#L30) are useless. Please let us know if you have not implemented it for a specific reason and it can be removed so we can submit a pull request for that.

Thanks for this useful component and I really appreciate any help you can provide.

**Comment by mrfelton** [comment]
Hi,
The original purpose of the MigrationMap collection was to enable you to store mappings between Object Ids pre and post migration which can be useful if you are migrating data from one system to another where Object Ids are being changed along the way. Arguably this is only useful for certain types of migrations and there are no useful tests for it.

I would consider a pull request that either adds a setting to make the creation of this model optional, or remove it completely from the component.
